src/BinSkim.Rules/RuleResources.resx

```diff
@@ -538,4 +538,13 @@ Modules triggering this check were:
   <data name="BA4002_ReportELFCompilerData_Description" xml:space="preserve">
     <value>This rule emits CSV data to the console for every compiler/language/version combination that's observed.</value>
   </data>
+  <data name="BA2026_EnableAdditionalSecurityChecks_Description" xml:space="preserve">
+    <value>/sdl enables a superset of the baseline security checks provided by /GS and overrides /GS-. By default, /sdl is off. /sdl- disables the additional security checks.</value>
+  </data>
+  <data name="BA2026_Error" xml:space="preserve">
+    <value>'{0}' does not enable the recommended Security Development Lifecycle (SDL) checks. To Enable the recommended Security Development Lifecycle (SDL) checks pass /sdl</value>
```

Comment:
>pass [](http://example.com/codeflow?start=164&length=4)

to  the linker? to the command line? where?

Reply:
fixed, copied from existing ones:
"pass ..... on the cl.exe command-line"